testsuite/src/test/groovy/noe/tomcat/BindingsTomcatConfiguratorIT.groovy

```diff
@@ -242,6 +245,29 @@ abstract class BindingsTomcatConfiguratorIT extends TomcatTestAbstract {
     assertEquals Tomcat.DEFAULT_HTTP_PORT + testOffset, Integer.valueOf(Server.Service.Connector.find { isNotSecuredHttpProtocol(it) }.@port.toString())
   }
 
+  @Test
+  void testCertificateDefaultsServerXmlChangeExpected() {
+    String sslStringDir = new File(new Platform().getTmpDir(), new PathHelper().join("ssl", "self_signed")).getCanonicalPath()
+    String sslCertificate = new File(sslStringDir, "server.crt").getCanonicalPath()
+    String sslCertificateKey = new File(sslStringDir, "server.key").getCanonicalPath()
+    String keystoreFilePath = new File(sslStringDir, "server.jks").getCanonicalPath()
+    String password = "changeit"
+    Integer testHttpsPort = 8443
+
+    new TomcatConfigurator(tomcat)
```

Comment:
please configure your IDE to use the same formatting as in `noe-core` (we have indentation 2 spaces).

Reply:
and update indentation where applicable in a new code

Reply:
done